src/meta/dashboard/src/proxy.rs

```diff
@@ -87,8 +88,17 @@ async fn proxy(
     let content = reqwest::get(url.clone()).await?;
 
     let resp = CachedResponse {
-        code: content.status(),
-        headers: content.headers().clone(),
+        code: hyper::StatusCode::from_u16(content.status().as_u16()).unwrap(),
+        headers: content.headers().iter().fold(
+            hyper::HeaderMap::with_capacity(content.headers().len()),
+            |mut header, it| {
+                header.insert(
+                    hyper::header::HeaderName::from_str(it.0.as_str()).unwrap(),
+                    hyper::header::HeaderValue::from_bytes(it.1.as_bytes()).unwrap(),
+                );
+                header
+            },
+        ),
         body: content.bytes().await?,
```

Comment:
This is needed because I had to downgrade `reqwest` to 0.11 to match the `deno_http` version (due to the recent axum version update)

Reply:
I ended up patching deno to use request 0.12 instead of changing this file

Reply:
But it seems to bring more submodules even if the feature is not enabled. 😕
```
+ cargo build -p risingwave_cmd_all --profile dev --features rw-static-link
    Updating git repository `https://github.com/bakjos/deno`
    Updating git submodule `https://github.com/denoland/deno_std`
    Updating git submodule `https://github.com/web-platform-tests/wpt.git`
    Updating git submodule `https://github.com/denoland/node_test.git`
```

Reply:
> I had to downgrade `reqwest` to 0.11 to match the `deno_http` version (due to the recent axum version update)

Can we add `reqwest 0.11` a dependency under a different name? Different versions should work together without issues.

Reply:
It's due to a version conflict of an inner dependency. 😕

```
error: failed to select a version for `hyper-util`.
    ... required by package `deno_http v0.141.0`
    ... which satisfies dependency `deno_http = "^0.141.0"` of package `arrow-udf-js-deno-runtime v0.0.1 (https://github.com/risingwavelabs/arrow-udf.git?rev=4c4e176#4c4e1762)`
    ... which satisfies git dependency `arrow-udf-js-deno-runtime` (locked to 0.0.1) of package `arrow-udf-js-deno v0.0.1 (https://github.com/risingwavelabs/arrow-udf.git?rev=4c4e176#4c4e1762)`
    ... which satisfies git dependency `arrow-udf-js-deno` (locked to 0.0.1) of package `risingwave_expr v1.7.0-alpha (/Users/wangrunji/Codes/risingwave/src/expr/core)`
    ... which satisfies path dependency `risingwave_expr` (locked to 1.7.0-alpha) of package `risingwave_cmd_all v1.7.0-alpha (/Users/wangrunji/Codes/risingwave/src/cmd_all)`
versions that meet the requirements `=0.1.2` are: 0.1.2

all possible versions conflict with previously selected packages.

  previously selected package `hyper-util v0.1.3`
    ... which satisfies dependency `hyper-util = "^0.1.3"` (locked to 0.1.3) of package `reqwest v0.12.2`
    ... which satisfies dependency `reqwest = "^0.12.2"` (locked to 0.12.2) of package `risedev v1.7.0-alpha (/Users/wangrunji/Codes/risingwave/src/risedevtool)`

failed to select a version for `hyper-util` which could resolve this conflict
```

Reply:
> It's due to a version conflict of an inner dependency. 😕

Do we have any idea why the upstream pins `hyper-util` with `=`?

Reply:
I think will pull the modules, but not compile them, I also patched the deno dependencies to use request 0.12, if you pull the latest commit, should compile ok